int64 front_left
int64 front_right
int64 front_left_mid
int64 front_right_mid
int64 depo_front
int64 depo_back
